libs/lib-client/activity-icon/src/activity-icon.component.ts

```diff
@@ -1,4 +1,10 @@
-import { Component, Input, OnChanges, ChangeDetectionStrategy } from '@angular/core';
+import {
```

Comment:
This is actually a new file that I copied from another file. You can disregard the older file at the left.